java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -2021,14 +2027,39 @@ protected void targetWindowClosingEvent(java.awt.event.WindowEvent e) {
      * @param inValidateMode boolean: if true, valid text == green, invalid text
      *                       == red background; if false, valid text == green,
      *                       invalid text == yellow background
-     * @param inEnable       boolean to enable / disable the JmriBeanComboBoxes
+     * @param inEnable       boolean to enable / disable the JmriBeanComboBox
      */
     public void setupComboBox(JmriBeanComboBox inComboBox, boolean inValidateMode, boolean inEnable) {
         inComboBox.setEnabled(inEnable);
         inComboBox.setEditable(true);
         inComboBox.setValidateMode(inValidateMode);
         inComboBox.setText("");
+        int c = Math.max(8, inComboBox.getItemCount());
```

Comment:
64 is too large. 16 is more appropriate maximum. My MacBookPro (13") can only display 43 items in a JComboBox containing 131 items, resulting in items 120-131 being unelectable because that is forcibly off screen.

I am certain there are users with smaller displays than I have.